team/482451/montyhall.py

```diff
@@ -0,0 +1,30 @@
+from random import randint
+
+c = 0
+list2 = []
+
+while c != 100000:
```

Comment:
If the number of iterations is fixed, a for loop is preferable.

Reply:
Good point. Always more readable.